src/components/DiplayBooks.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState, useEffect } from 'react';
+import ImageList from '@mui/material/ImageList';
+import ImageListItem from '@mui/material/ImageListItem';
+import ImageListItemBar from '@mui/material/ImageListItemBar';
+import SearchBar from './SearchBar';
+import TextField from "@mui/material/TextField";
+
+function DiplayBooks() {
+  const [error, setError] = useState(null);
+    const [isLoaded, setIsLoaded] = useState(false);
+    const [books, setBooks] = useState([]);
+    const [records, setRecords]=useState(books);
+    const Filter= (event)=>{
+        setRecords(books.filter(f=>(f.title.toLowerCase().includes(event.target.value.toLowerCase()) 
+        ||(f.author.toLowerCase().includes(event.target.value.toLowerCase()) ) )))
+    }
+    
+      
+    useEffect(() => {
+        fetch('http://localhost:8080/Home')
```

Comment:
replace the fetch URL with 'http://localhost:8080/api/books/home'

Reply:
Make sense, I will update that in Display.jsx file, thanks!

Reply:
Can anyone of you please reject my front-home branch merge request, thanks!